### ISSUE TITLE
`eliminate` creates duplicate HalfSpaces

### ISSUE THREAD

**Issue by hdavid16** [JuliaPolyhedra/Polyhedra.jl#307]
Is it possible to remove duplicates in the output of `eliminate`? I notice that eliminating often creates duplicate HalfSpaces in the iterator of the resulting Polyhedron. I think this is really hurting performance when eliminating many dimensions...

**Comment by blegat** [comment]
Yes, you can remove it with [`removehredundancy`](https://juliapolyhedra.github.io/Polyhedra.jl/stable/redundancy/#Polyhedra.removehredundancy!)
